content-webapp/src/main/webapp/vue-app/news/components/ExoNewsDetailsActionMenuApp.vue

```diff
@@ -120,12 +120,18 @@ export default {
   computed: {
     isMobile() {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
+    },
+    scheduled() {
+      return !!this.news.schedulePostDate || this.staged;
+    },
+    staged() {
+      return this.news?.publicationState === 'staged';
     }
```

Comment:
which diff between scheduled and staged ?

Reply:
> which diff between scheduled and staged ?

To identify the draft of scheduled article since its publicationState is "draft",  and keep the choose saved schedule properties after editing a scheduled article without cancelling it